tgui/packages/tgui-say/handlers/incrementChannel.tsx

```diff
@@ -1,32 +1,49 @@
 import { CHANNELS } from '../constants';
 import { Modal } from '../types';
+// Insert the names of channels you want to not cycle on tab here
+const BLACKLIST = ['Admin'];
+const BLACKLISTED_CHANNELS = [] as number[];
+CHANNELS.forEach((channel, index) => {
+  if (BLACKLIST.includes(channel)) {
+    BLACKLISTED_CHANNELS.push(index);
+  }
+});
```

Comment:
Change this whole thing into a map, e.g. `BLACKLIST.map((blacklistedChannel) => CHANNELS.findIndex(blacklistedChannel)` or whatevs, but not this

Also you should change to BLACKLISTED_CHANNEL_INDICES or similar, because right now this is just a synonym

Reply:
Alright, done